src/style.css

```diff
@@ -1,3 +1,89 @@
-body {
+/* #region ZERO */
+body{
+  padding: 0;
   margin: 0;
+  font-family: Roboto, sans-serif;
+  padding-bottom: 2000px;
+}
+
+.wrapper{
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+:root{
+  --header-height: 60px;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+/* #endregion ZERO */
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  object-fit: contain;
+  padding: 10px 0;
+}
+
+nav{
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to style elements using classes to ensure that the styles are not too broadly applied and can be reused more effectively.